bin/failedTestLogs.sh

```diff
@@ -1,28 +1,30 @@
 #!/bin/sh
```

Comment:
I think things like ${FILE##*.} are not in the basic Bourne shell, so let's make this /bin/bash